hashes/src/sha256.rs

```diff
@@ -114,7 +114,7 @@ impl Hash {
     /// Computes hash from `bytes` in `const` context.
     ///
     /// Warning: this function is inefficient. It should be only used in `const` context.
-    pub const fn const_hash(bytes: &[u8]) -> Self { Hash(Midstate::const_hash(bytes, true).0) }
+    pub const fn hash_unoptimized(bytes: &[u8]) -> Self { Hash(Midstate::hash_unoptimized(bytes, true).0) }
```

Comment:
Deprecate? Although it's probably not used much.

Reply:
Is it worth it with all this re-writing going on?

Reply:
Unlikely in this case.